src/CSFML/Graphics/TextureStruct.hpp

```diff
@@ -35,22 +35,14 @@
 ////////////////////////////////////////////////////////////
 struct sfTexture
 {
-    sfTexture()
-    {
-        This        = new sf::Texture;
-        OwnInstance = true;
-    }
+    sfTexture() = default;
 
-    sfTexture(sf::Texture* texture)
+    sfTexture(sf::Texture* texture) : This(texture), OwnInstance(false)
     {
-        This        = texture;
-        OwnInstance = false;
     }
 
-    sfTexture(const sfTexture& texture)
+    sfTexture(const sfTexture& texture) : This(texture.This ? new sf::Texture(*texture.This) : nullptr)
     {
-        This        = texture.This ? new sf::Texture(*texture.This) : nullptr;
-        OwnInstance = true;
```

Comment:
I don't know what `OwnInstance` is used for, but this changes semantics, as the `sf::Texture*` constructor overload will set `OwnInstance` to `false`, while this constructor is supposed to set it to `true`.

Reply:
`OwnInstance` determines whether `delete` is called on `This`. I changed the default value of `OwnInstance` to `true` so its value in this constructor shouldn't be different than before.

Reply:
Ah right, I misread that